src/main/features/dashboard/views/macro/claimStatus/claimFullAdmission.njk

```diff
@@ -0,0 +1,116 @@
+    <p> {{ t('If you pay by cheque or transfer the money must be clear in their account.') }} </p>
+    <p> {{ t('If they don’t receive the money by then, they can request a County Court Judgment against you.') }} </p>
+    <p> {{ t('Contact {{ claimantName }} if you need their payment details.',
+        { claimantName: claimantName }) }} </p>
+    <p> {{ t('Make sure you get receipts for any payments.') }} </p>
+
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    <p> {{ t('You’ve offered to pay {{ claimantName }} {{ amount }} on {{ paymentDate }}.',
+        { claimantName: claimantName, amount: amount, paymentDate: paymentDate }) }} </p>
+    <p>
+      {{ t('We’ll contact you when they respond.') }}
+    </p>
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    {% if claim.response.paymentIntention.repaymentPlan.paymentSchedule === domain.PaymentSchedule.EVERY_MONTH %}
+      {% set paymentSchedule = 'per month' %}
+    {% elif claim.response.paymentIntention.repaymentPlan.paymentSchedule === domain.PaymentSchedule.EVERY_TWO_WEEKS %}
+      {% set paymentSchedule = 'per two weeks' %}
+    {% elif claim.response.paymentIntention.repaymentPlan.paymentSchedule === domain.PaymentSchedule.EACH_WEEK %}
+      {% set paymentSchedule = 'per week' %}
+    {% endif %}
+    {% set repaymentDate = claim.response.paymentIntention.repaymentPlan.firstPaymentDate |date %}
+    <p> {{ t('You’ve offered to pay {{ claimantName }} {{ amount }} {{ paymentSchedule }} starting {{ repaymentDate }}.',
+        { claimantName: claimantName, amount: amount, paymentSchedule: paymentSchedule, repaymentDate: repaymentDate }) }} </p>
+    <p>
+      {{ t('We’ll contact you when they respond.') }}
+    </p>
+  {% endif %}
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimFullAdmissionForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+
+  {% set claimantName = claim.claimData.claimant.name %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {% set amount = claim.totalAmountTillToday | numeral %}
+
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    <h2 class="heading-small">{{ t('The defendant will pay you immediately') }}</h2>
+    <p> {{ t('The defendant said they’ll pay you immediately. They must make sure you have the money by 4pm on {{ paymentDate }}.',
+        { paymentDate: paymentDate }) }} </p>
+    <p> {{ t('You need to tell us if you’ve settled the claim because the defendant has paid you.') }} </p>
+    <p> {{ t('You can settle for less than the full claim amount.') }} </p>
+    {% if claim.pastDefendantPayImmediatelyDate %}
```

Comment:
As pointed earlier please reimplement it so that the method is removed from claim class.